docs/api/geometries/PlaneGeometry.html

```diff
@@ -52,8 +52,9 @@ <h3>[name]([param:Float width], [param:Float height], [param:Integer widthSegmen
 
 		<h2>Properties</h2>
 
+		<h3>.parameters</h3>
```

Comment:
Is there are reason why you have added this line on in certain files? It would be consistent if you apply this change to all geometry pages. Same for the subsequent description.

Reply:
I've made it more consistent now. Also for the subsequent description. From cursory browsing `.properties` is mentioned in the other Geometry classes, in a slightly different way. I just found + replaced the ones that were all using the same wording.

Reply:
e.g. https://threejs.org/docs/index.html#api/geometries/BoxBufferGeometry

Reply:
There are still pages like `CircleGeometry` that don't have any docs about their parameters. Besides, the formatting of `parameters` should be similar to the one in `DodecahedronGeometry`.
```html
<h3>[property:Object parameters]</h3>
```
In this way it is more consistent to the rest of the docs.